src/bucketizers/timebasedBucketizer.ts

```diff
@@ -143,15 +146,17 @@ export default class TimebasedBucketizer implements Bucketizer {
                         parseInt(bucketProperties[1]) / this.k <
                         this.minBucketSpan
                     ) {
-                        console.log("We need to make a new page");
```

Comment:
I would keep these. Is helpful when debugging. But in #40 they are properly logged only in debug mode.